inc/saiswitch.h

```diff
@@ -3048,6 +3048,33 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_MAX_ICMP_ECHO_SESSION,
 
+    /**
+     * @brief Maximum size of the packet after trimming
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 128
+     */
+    SAI_SWITCH_ATTR_PACKET_TRIMMING_SIZE,
```

Comment:
@marian-pritsak 
We need to also clarify that trimming a packet MUST update the IP length and IP checksum.
L4 is not touched i.e. for eg UDP length and checksum is not updated

Reply:
Added this comment to the action description in saibuffer.h